test/js/fields/ArrayField.unit.spec.jsx

```diff
@@ -89,12 +89,59 @@ describe('Schemaform <ArrayField>', () => {
         formData={formData}
         formContext={formContext}
         onChange={f => f}
-        requiredSchema={requiredSchema}/>
+        requiredSchema={requiredSchema}
+        errorSchema={[]}/>
     );
 
     expect(tree.everySubTree('SchemaField').length).to.equal(1);
     expect(tree.everySubTree('.va-growable-background').length).to.equal(2);
   });
+  it('should render invalid items', () => {
+    const idSchema = {
+      $id: 'field'
+    };
+    const schema = {
+      type: 'array',
+      items: [],
+      additionalItems: {
+        type: 'object',
+        properties: {
+          field: {
+            type: 'string'
+          }
+        }
+      }
+    };
+    const uiSchema = {
+      'ui:title': 'List of things',
+      'ui:options': {
+        viewField: f => f
+      }
+    };
+    const formData = [
+      { field: true },
+      {}
+    ];
+    const errorSchema = [
+      { field: { __errors: ['Invalid type'] } }
+    ];
+    const tree = SkinDeep.shallowRender(
+      <ArrayField
+        schema={schema}
+        uiSchema={uiSchema}
+        idSchema={idSchema}
+        registry={registry}
+        formData={formData}
+        formContext={formContext}
+        onChange={f => f}
+        requiredSchema={requiredSchema}
+        errorSchema={errorSchema}/>
+    );
+
+    // First SchemaField is the invalid item, second is normally in edit mode
+    expect(tree.everySubTree('SchemaField').length).to.equal(2);
```

Comment:
So if you look at the test before this one, you'll see the same `expect` statements, only with `SchemaField` found only once. I think this is because `SchemaField` is found when an item is in edit mode and the last item in the array is always in edit mode unless the `reviewMode` option is set to true. (:thinking: or...I don't remember what exactly it's called off the top of my head.)

In this case, that means the first item is invalid and--with these changes--should be in edit mode, and the last one is also in edit mode because I didn't set that `review` option.

Mind you, I didn't verify this, but I'm pretty sure that's what's going on. :crossed_fingers: